csm/conf/etc/csm/csm.conf

```diff
@@ -133,6 +133,7 @@ SUPPORT_BUNDLE:
     symlink_path: "/tmp/support_bundle/"
     cluster_file_path : "/opt/seagate/cortx/provisioner/pillar/components/cluster.sls"
     ssh_user : "root"
+    alerts_filename: "fru_alerts.json"
```

Comment:
Why is it called have fru_alerts.json ? This is going to have all kinds of alerts.

Reply:
Ok will change it

Reply:
Done